hathor/sysctl/protocol.py

```diff
@@ -12,138 +12,30 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import inspect
-import json
-from typing import TYPE_CHECKING, Any, Callable, Optional
-
-from pydantic import ValidationError
 from twisted.protocols.basic import LineReceiver
 
-from hathor.sysctl.exception import SysctlEntryNotFound, SysctlException, SysctlReadOnlyEntry, SysctlWriteOnlyEntry
-
-if TYPE_CHECKING:
-    from hathor.sysctl.sysctl import Sysctl
+from hathor.sysctl.runner import SysctlRunner
 
 
 class SysctlProtocol(LineReceiver):
     delimiter = b'\n'
 
-    def __init__(self, root: 'Sysctl') -> None:
-        self.root = root
+    def __init__(self, runner: SysctlRunner) -> None:
+        self.runner = runner
 
     def lineReceived(self, raw: bytes) -> None:
         try:
             line = raw.decode('utf-8').strip()
         except UnicodeDecodeError:
             self.sendError('command is not utf-8 valid')
-        if line.startswith('!help'):
```

Comment:
Why don't we keep custom commands `!help` and `!backup` here? I'm not sure that we should move them to the runner.

What do you think @jansegre ?

Reply:
`!help` and `!backup` in particular seem to be useful only for an interactive CLI, I don't think there's too much benefit in having them available directly in the runner, so I agree with @msbrogli that maybe it makes more sense to leave them here.

Reply:
Done: 487964a80dbc2feaac3b0aa40f4d1a21c6821208

Reply:
Thanks!